.github/workflows/build.yml

```diff
@@ -33,5 +33,6 @@ jobs:
             go.sum
             *.toml
       - run: |
-          make build
```

Comment:
Why is this change needed?

Reply:
Good catch @hanchon, having a new target in the Makefile and a step in the workflow that support `pebbleDB` should be enough 👍 
Reverting.